features/step_definitions/login_steps.rb

```diff
@@ -1,14 +1,25 @@
 Given(/^the login page$/) do
-  #put your code here
-  pending
+  visit("/")
 end
 
 When(/^I log in with proper credentials$/) do
-  #put your code here
-  pending
+  fill_in("username", :with => "User")
```

Comment:
use new syntax with: instead of :with => also the ' is preferred over " if you don't have expressions inside the string.